compute/instances/how-to/create-image-from-snapshot.mdx

```diff
@@ -35,3 +36,54 @@ categories:
 <Message type="tip">
   You can also create an image from the **Snapshots** dashboard by clicking <Icon name="more"/> next to the snapshot from which you want to create an image. Click **Image from snapshot**. A new tab opens: enter a name for your image. Then click **Create image from snapshot**.
 </Message>
+
+## Creating an image via the Scaleway CLI
+
+You can also create an image using the [Scaleway CLI](/developer-tools/scaleway-cli/quickstart/). Use the following command:
+
+```bash
+scw instance image create snapshot-id="$SCW_VOLUME_ID_1" additional-volumes.0.id="$SCW_VOLUME_ID_2" arch="x86_64"
+```
+
+- Replace `$SCW_VOLUME_ID_1` with the ID of the snapshot or volume you want to use as the root volume.
+- Replace `$SCW_VOLUME_ID_2` with the ID of any additional volume to be included.
+- Ensure the architecture (`arch`) is set to a valid value (e.g., `x86_64`).
+
+<Message type="note">
+  Supported volume IDs include:
+  - Local SSD (`l_ssd`)
+  - Block SSD (`b_ssd`)
+  - Block Storage Low Latency (`sbs`) volumes.
+</Message>
+
+## Creating an image via the Scaleway API
+
+You can also create an image using the [Scaleway API](https://www.scaleway.com/en/developers/api/instance/#path-images-create-an-instance-image). Use the following `curl` command:
+
+```bash
+curl --location "https://api.scaleway.com/instance/v1/zones/$SCW_AVAILABILITY_ZONE/images" \
+--header "Content-Type: application/json" \
+--header "X-Auth-Token: $SCW_SECRET_KEY" \
+--data '{
+    "root_volume": "'$SCW_VOLUME_ID_1'",
+    "arch": "x86_64",
+    "name": "image_name",
+    "extra_volumes": {
+        "1": {"id":"'$SCW_VOLUME_ID_2'"}
+    },
+    "organization": "'$SCW_ORG_ID'"
+}'
+```
+
+- Replace `$SCW_VOLUME_ID_1` with the ID of the snapshot or volume to use as the root volume.
+- Replace `$SCW_VOLUME_ID_2` with the ID of any additional volume.
+- Use `$SCW_ORG_ID` for the ID of your organization.
+- Ensure the architecture (`arch`) is set to a valid value (e.g., `x86_64`).
+
+<Message type="tip">
+  Supported volume IDs include:
+  - Local SSD (`l_ssd`)
```

Comment:
Maybe use the names we decided on during the wording session? (Local Storage & Block Storage legacy)